src/eterna/mode/FeedbackViewMode.ts

```diff
@@ -73,22 +65,21 @@ export default class FeedbackViewMode extends GameMode {
             if (Eterna.MOBILE_APP) {
                 window.frameElement.dispatchEvent(new CustomEvent('navigate', {detail: '/'}));
             } else {
-                window.location.href = EternaURL.createURL({page: 'lab_bench'});
+                window.location.href = EternaURL.createURL({page: 'home'});
             }
         });
         this.addObject(this._homeButton, this.uiLayer);
 
         const homeArrow = new Sprite(BitmapManager.getBitmap(Bitmaps.ImgHomeArrow));
         homeArrow.position = new Point(45, 14);
-        Assert.assertIsDefined(this.container);
-        this.container.addChild(homeArrow);
+        this.uiLayer.addChild(homeArrow);
 
-        const puzzleTitle = UITheme.makeTitle(this._puzzle.getName(true), 0xffffff);
+        const puzzleTitle = UITheme.makeTitle(this._puzzle.getName(!Eterna.MOBILE_APP), 0xffffff);
```

Comment:
aha, I assume this is related. we didn't need it after all

Reply:
This is actually a change I didn't realize was made due to a copy-paste, but if we ever implement this into the mobile app (who knows!) it'll be necessary

Reply:
Oh, I see you were referring to the previous comment - yup